src/tools/annotation/CobbAngleTool.js

```diff
@@ -405,7 +405,7 @@ export default class CobbAngleTool extends BaseAnnotationTool {
 
     const suffix = !rowPixelSpacing || !colPixelSpacing ? ' (isotropic)' : '';
 
-    return `${toGermanNumberStringTemp(rAngle)}\u00B0${suffix}`;
+    return `${localization.localizeNumber(rAngle)}\u00B0${suffix}`;
```

Comment:
This is the same as the angle tool, should we consider "isotropic" in our localisation?

Reply:
@LisaFrauenstein ?